### ISSUE TITLE
Improve overall UI design, readability, and UX

### ISSUE THREAD

**Issue by bnvk** [schildbach/bitcoin-wallet#290]
Currently, the Android app feels a bit rough around the edges in terms of interface design and usability. The basics work, but there are many subtle things (which are not terribly work intensive) that I think will improve the app's UX a bit. Some of these are:

- Typography, boldenings, headings in views with lots of text (Settings)
- Information architecture of items the "..." menu (Safety, Settings, etc...)
- Iconography of items in menu

I'm happy to learn some light Android dev and / or provide screenshots of what I'm thinking along these lines, as long these changes are going to be welcome from core devs for discussion and the possibility of accepting pull requests!

**Comment by schildbach** [comment]
Yes, we do welcome concrete examples or – even better – pull requests.

**Comment by franckuestein** [comment]
Hi @bnvk, feel free to provide some mockups and samples to see what you're thinking about 😉

**Comment by bnvk** [comment]
@schildbach @franckuestein glad you're open to this. I will do so in the coming weeks. Kind of slammed at the moment, but I will :smile:

**Comment by DoWhileGeek** [comment]
Has anyone considered redesigning the app using Google's Material UI guidelines? They make for gorgeous apps, and match the android ecosystem.

**Comment by NQAT** [comment]
I don't mind the layout, but I think a new color scheme or theme would go a long ways and give it new feel. Maybe a version without grey tones and dark menu bars. I'm not familiar with android apps though.

**Comment by franckuestein** [comment]
I like the current one, but after a bit of free time I've been looking for some basis to start working on a design adapted to Material Design styles for Bitcoin Wallet.

![Material design wallet](https://cdn.pbrd.co/images/QyUjQWU.png)

The idea is to make the fewest possible changes but noticeable enough to be a good redesign.
What do you think about it?

I'm not a professional graphical designer, so after looking for "free wallet psd's" I found the [Axis Material Concept](https://dribbble.com/shots/2085584-Axis-Google-material-Concept) free .psd by samsu on dribbble that's basically taking the new menus, navbars, buttons... styles. IMHO it could perfectly fit on a Material Design redesign for Schildbach wallet.

Hope you like it! You'll notice that there are just 3 app screenshots, if you like It I can work on the other ones and show it to you. 😉

**Comment by schildbach** [comment]
Thanks for your ideas! It's interesting to see how the orange title/action bar works out.

Unfortunately, the main screen is *much* more complex than what's seen on the screenshot. There can be a lot more messages, both at the bottom of the screen plus for each transaction. Transactions have context menus, the wallet itself has a context menu (which I think cannot/should not completely go into the navigation drawer). Transactions have a lot of different confidence states (most notably the number of blockchain confirmations). The QR code is missing, and the address book is not comparable to the filters (all/sent/received). And maybe most important, we need variants for landscape orientation, large screens, etc.

Some additional notes: The app – by design – does not come with a splash screen and doesn't need a "close wallet". Can you also remove references to "Schildbach wallet"? The app is developed by a group of people best described as "Bitcoin Wallet developers".

**Comment by DoWhileGeek** [comment]
There should not be three or more buttons for basic actions on the bottom left, it should be one plus button that exposes more, as defined in the material style guidelines. Also, we as a society are thoroughly done with gaussian blur, 2007 has come and gone.

Sorry for the harshness, but I care about the design of this app, and I dont want to see it stray farther from material design, rather than towards.

**Comment by AyeSarita** [comment]
Is a very simple clean layout. ..

**Comment by AyeSarita** [comment]
Are you looking for a design that a bit bold but business professional..
???

**Comment by schildbach** [comment]
The app is now using material theme on Lollipop+.
